axolotl-web/src/pages/About.vue

```diff
@@ -4,11 +4,16 @@
     <h1 class="title">Axolotl Beta {{config.Version}}</h1>
     <h2 class="subtitle" v-translate>A cross-plattform signal client</h2>
     <div class="description">
-      <span v-translate>This is a free and open source signal client written in golang and vuejs.
-      You can support the development of axolotl either by filling</span> <br/>
-      <a href="https://github.com/nanu-c/axolotl/issues" @click="openExtern($event, 'https://github.com/nanu-c/axolotl/issues')" ><span v-translate>issues at the bug tracker</span></a>. <br />
-      <span v-translate>or</span><br />
-      <span v-translate>by becoming a </span>
+      <span v-translate class="mr-1">This is a free and open source signal client </span>
+      <span v-translate class="mr-1">written in golang and vuejs.</span>
+      <br />
+      <span v-translate>You can support the development of axolotl either by filling</span>
+      <br/>
+      <a href="https://github.com/nanu-c/axolotl/issues" @click="openExtern($event, 'https://github.com/nanu-c/axolotl/issues')" ><span v-translate>issues at the bug tracker</span></a>.
+      <br />
+      <span v-translate>or</span>
+      <br />
+      <span v-translate class="mr-1">by becoming a</span>
```

Comment:
Maybe that can be done later, but I have the feeling we're not using the translation tool correctly. To me, the whole sentence should always be in only one `v-transtlate`. It's not a problem if there are br's in it, the translators will copy them. If it's more complicated, like links, then we need to use %{} to insert the link, but the sentence still have to be one translation so it is understandable by the translators.